manifests/yamls/lens.yaml

```diff
@@ -182,7 +182,7 @@ spec:
     app: lens
 ---
 # Source: lens/templates/hpa.yaml
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2beta2
```

Comment:
we should use capabilties to ensure hpa is compatable in older versions

Reply:
added support for older hpa apiversion